tests/README.md

```diff
@@ -214,16 +179,22 @@ make test.kube.trafficmanagement
 
 1. If HUB and TAG aren't set, then their default values will match what is used by the prow tests.
 
-## Running Tests: go test
+## Migrate from Old Framework
```

Comment:
I don't think we want to document migration instructions. As soon as the new framework is ready, we can do a one-time conversion of the existing 22 tests in a single PR, and also in the same PR remove the old framework code.

Reply:
Sounds good! I'll remove this part